assembl/fabfile.py

```diff
@@ -2132,6 +2132,56 @@ def set_ssl_certificates():
         print(yellow("Can't set ssl certificates, env.ocsp_path is not set"))
 
 
+@task
+def create_backup_script():
+    """
+    Generates backup script that stores the backup on a local borg repository.
+    Sets a cron job for it.
+    """
+    from jinja2 import Environment, FileSystemLoader
+    print(green("Generating backup script on remote host."))
+    jenv = Environment(loader=FileSystemLoader('doc/'), autoescape=lambda t: False)
+    backup_template = jenv.get_template('backup_template.jinja2')
+    with open('backup_all_assembl.sh', 'w') as f:
+        f.write(backup_template.render(user=env._user, project_path=env._projectpath))
+    put('backup_all_assembl.sh', '/home/assembl_user/backup_all_assembl.sh')
+    run('chmod +x backup_all_assembl.sh')
+    cron_command = "15 3 * * * /home/" + env._user + "/backup_all_assembl.sh"
+    create_add_to_crontab_command(cron_command)
+
+
+@task
+def create_alert_disk_space_script():
+    """Generates the script to alert on disk space limit and sets cron job for it."""
+    from jinja2 import Environment, FileSystemLoader
+    jenv = Environment(loader=FileSystemLoader('doc/'), autoescape=lambda t: False)
+    alert_disk_space_template = jenv.get_template('alert_disk_space_template.jinja2')
+    with open('alert_disk_space.sh', 'w') as f:
+        f.write(alert_disk_space_template.render())
+    put('alert_disk_space.sh', '/home/assembl_user/alert_disk_space.sh')
+    run('chmod +x alert_disk_space.sh')
+    cron_command = "MAILTO=assembl.admin@bluenove.com"
```

Comment:
Should also be an rc value, defaults to this address.

Reply:
done